pkg/devenvutil/devenv.go

```diff
@@ -361,6 +363,54 @@ func WaitForAllPodsToBeReady(ctx context.Context, k kubernetes.Interface, log lo
 	return nil
 }
 
+func deleteBackoffPods(ctx context.Context, k kubernetes.Interface, log logrus.FieldLogger, pods []*corev1.Pod) {
+	backoffPods := filter(pods, func(p *corev1.Pod) bool {
+		_, _, isCrashLoopBackoff := debug.ProblemPodCrashLoopBackOff.Detector(ctx, p)
+		return isCrashLoopBackoff
+	})
+
+	if len(backoffPods) == 0 {
+		return
+	}
+	podNames := mapTo(backoffPods, func(p *corev1.Pod) string { return p.Name })
+	log.Infof("Deleting pods to speed things up: %s", strings.Join(podNames, ", "))
+	if err := deletePodsSync(ctx, k, backoffPods); err != nil {
+		log.Warnf("Unable to delelete crashing pods: %s", err)
+	}
+}
+
+func deletePodsSync(ctx context.Context, k kubernetes.Interface, podsToDelete []*corev1.Pod) error {
+	const gracePeriod int64 = 0
```

Comment:
nit: `const` should generally only ever be used for package level variables, also use the default value when able:

```suggestion
	var gracePeriod int64
```